src/Sylius/Behat/Context/Domain/ManagingOrdersContext.php

```diff
@@ -147,6 +147,18 @@ public function thisOrderHasNotBeenPaidForDays(OrderInterface $order, $amount, $
         $this->unpaidOrdersStateUpdater->cancel();
     }
 
+    /**
+     * @Given /^the order with number "([^"]*)" has not been paid for (\d+) (day|days)$/
+     */
+    public function orderWithNumberHasNotBeenPaidForDays(string $orderNumber, $amount, $time)
```

Comment:
If I see correctly, `$time` is a string that can have value: `day` or `days`, it is totally confusing to me. You should probably change the name of this variables or combine them to one as there are used only in place together. And remember about types of arguments and return type, because you missed that here